roadmaps/windows.md

```diff
@@ -111,8 +130,11 @@ GitHub](https://github.com/bazelbuild/bazel-website/commits/master/roadmaps/wind
         ([#3888](https://github.com/bazelbuild/bazel/issues/3888)).
 
 *   **Fix the following set of uncategorized issues.**
-    *   **[scheduled: Q2'18]** Write Skylark rule migration guide
+    *   **[on hold]** Write Skylark rule migration guide
         ([#3889](https://github.com/bazelbuild/bazel/issues/3889)).
+        *   **[status as beginning of Q3'18]**: Was scheduled for Q2'18 but we
+            couldn't deliver it. We need to put this on hold because the other
+            scheduled work is more important.
```

Comment:
I think we should work on this sooner rather than later. Bazel without rules is not usable, and on Windows too.

Reply:
I agree this is important, alas we have no people to work on it.
Otherwise other stuff has to wait.